src/rajAJun20/ReplaceAndFindIndexOfStringUsingArrayList.java

```diff
@@ -0,0 +1,46 @@
+/*Program 2 :
+Display on which index Techno is available, Replace all Techno with Credits from given ArrayList, and count total number of Credits.
+["Maulik","Techno" ,"Krishna", "Techno", "Credits", "Techno"]
+Output : 
+Techno word index in given arrayList is 1,3,5 
+["Maulik","Credits" ,"Krishna", "Credits", "Credits", "Credits"]
+Total Credits words :- 4 */
+
+
+package rajAJun20;
+
+import java.util.ArrayList;
+
+public class ReplaceAndFindIndexOfStringUsingArrayList {
+
+	void replaceAndFindIndexOfStringFromArrayList(ArrayList<String> inputString , String previousStr , String replaceWithNewStr) {
+		int count = 0 ;
+		System.out.print("Index value of string " + previousStr + " in given ArrayList is : ");
+		while(inputString.contains(previousStr)) {
+			int position = inputString.indexOf(previousStr);
+			System.out.print(position+ " ");
+			inputString.set(position , replaceWithNewStr);
+		}
```

Comment:
You can directly count credits after this.. There is no need of Second loop
while(inputString.contains(previousStr)) {
			int position = inputString.indexOf(previousStr);
			System.out.print(position+ " ");
			inputString.set(position , replaceWithNewStr);
                          count++;
}
System.out.println(Count);